Makefile

```diff
@@ -779,7 +779,7 @@ roxvet: $(ROXVET_BIN)
 	@# TODO(ROX-7574): Add options to ignore specific files or paths in roxvet
 	$(SILENT)go list -e ./... \
 	    | $(foreach d,$(skip-dirs),grep -v '$(d)' |) \
-	    xargs -n 1000 go vet -vettool "$(ROXVET_BIN)" -donotcompareproto -gogoprotofunctions -tags "sql_integration"
+	    xargs -n 1000 go vet -vettool "$(ROXVET_BIN)" -donotcompareproto -gogoprotofunctions -tags "sql_integration test_e2e"
```

Comment:
I think we need a separated run for all roxvet analyzers for this tag

Reply:
Having multiple go tags acts like an OR from my understanding, so now it should run both `sql_integration` and `test_e2e`. At least it was clearly linting the files I had tagged with `test_e2e` correctly when i ran roxvet locally.

What problem do you foresee with running it this way?

Reply:
> Do we need to add tags to golangci-lint?

As for this question, I think if we don't add the tag it will ignore files tagged test_e2e. Is that not also why it is tagged `sql_integration`?